tests/phpunit/tests/fonts/font-library/wpFontsDir.php

```diff
@@ -69,4 +69,25 @@ function set_new_values( $defaults ) {
 
 		$this->assertSame( static::$dir_defaults, $font_dir, 'The wp_get_font_dir() method should return the default values.' );
 	}
+
+	public function test_fonts_dir_with_uploads_default() {
+		$font_dir = wp_get_font_dir();
+		chmod( $font_dir['path'], 0000 );
```

Comment:
This line is not working as expected.

Reply:
Changing directory permissions isn't going to work. 

* `0000` will not be resettable during tear down, as it'll throw a permissions denied error.
* Setting `wp-content` to read-only => `is_rewritable()` returns `true`.
* Adding an existing `wp-content/fonts` and setting it to read-only (before testing) => `is_rewritable()` returns `true`.

Reply:
@costdev had an idea to create a new `wp-content/fonts` file (not directory) as a fake to trigger the fallback logic. It works. It's weird. But it works for what needs to get tested.